web/views/bundles_view.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Cog.V1.BundlesView do
+  use Cog.Web, :view
+
+  alias Ecto.Association
+
+  def render("bundle.json", %{bundle: bundle}) do
+    %{id: bundle.id,
+      name: bundle.name,
+      namespace: bundle.namespace.name,
+      commands: render_commands(bundle.commands),
+      relay_groups: render_relay_groups(bundle.relay_groups),
+      enabled: bundle.enabled,
+      inserted_at: bundle.inserted_at,
+      updated_at: bundle.updated_at}
+  end
+  def render("index.json", %{bundles: bundles}) do
+    %{bundles: render_many(bundles, __MODULE__, "bundle.json", as: :bundle)}
+  end
+  def render("show.json", %{bundle: bundle}) do
+    %{bundle: render_one(bundle, __MODULE__, "bundle.json", as: :bundle)}
+  end
+
+  def render_relay_groups(%Association.NotLoaded{}) do
```

Comment:
Doesn't this mean that we'll show different results to the user depending on which code path our data took to get here? Seems like we should always load the association if it's part of our API contract to show them.